website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -48,12 +48,16 @@ Attribute                        | Description
 ---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 `hosted-json-state-download-url` | A URL from which you can download the state data in a [stable format](/terraform/internals/json-format) appropriate for external integrations to consume. Only available if the state was created by Terraform 1.3+.
 `hosted-state-download-url`      | A URL from which you can download the raw state data, in the format used internally by Terraform.
+`hosted-json-state-upload-url`   | A URL to which you can upload the state data, in a [stable format](/terraform/internals/json-format) appropriate for external integrations to consume. You can only upload JSON state content once per state version.
+`hosted-state-upload-url`        | A URL to which you can upload the state data, in the format used internally by Terraform. You can only upload state data once per state version.
 `modules`                        | Extracted information about the Terraform modules in this state data. Populated asynchronously.
 `providers`                      | Extracted information about the Terraform providers used for resources in this state data. Populated asynchronously.
+`provisional`                    | A boolean flag that indicating that Terraform Cloud created a state version but has not yet marked as the current version for a workspace.
 `resources`                      | Extracted information about the resources in this state data. Populated asynchronously.
 `resources-processed`            | A Boolean flag indicating whether Terraform Cloud has finished asynchronously extracting outputs, resources, and other information about this state data.
 `serial`                         | The serial number of this state instance, which increases every time Terraform creates new state in the workspace.
 `state-version`                  | The version of the internal state format used for this state. Different Terraform versions read and write different format versions, but it only changes infrequently.
+`status`                         | Indicates a state version's content upload status. Can be `pending`, `finalized` or `discarded`.
 `terraform-version`              | The Terraform version that created this state. Populated asynchronously.
 `vcs-commit-sha`                 | The SHA of the configuration commit used in the Terraform run that produced this state. Only present if the workspace is connected to a VCS repository.
 `vcs-commit-url`                 | A link to the configuration commit used in the Terraform run that produced this state. Only present if the workspace is connected to a VCS repository.
```

Comment:
Duplicated from my PR #421 

```suggestion
`vcs-commit-url`                 | A link to the configuration commit used in the Terraform run that produced this state. Only present if the workspace is connected to a VCS repository.

### State Version States

The state version state is found in `data.attributes.status`, and you can reference the following list of possible states.

A state version created through the API or CLI will only be listed in the UI if it is in the `finalized` state.

| State        | Description                                                                                                                                                                                                                                                                                                                                                                                                                        |
| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `pending`    | The initial status of a state version after it has been created without encoding the state data within the request. Pending state versions do not contain state data and will not appear in the UI. The workspace cannot be unlocked normally until the latest state version has been finalized.                                     |
| `finalized`  | The state version has been uploaded successfully to Terraform Cloud or if the state version was created with a valid `state` attribute.                                                                                                                                                                                                              |
| `discarded` | The state version has been discarded because it was provisional and not promoted to be the current state version of the workspace. |
```

Reply:
Possibly silly question: Can we call them something besides "state version states"? Reading the first sentence is kinda confusing with the number of times the word "state" comes up.

Reply:
For sure. You are right about that. I got a little carried away! State Version Status would be the next best thing